src/components/Spells/index.jsx

```diff
@@ -3,24 +3,42 @@ import { WIZARD_WORLD_BASE_URL } from "../../constants";
 import { useDispatch, useSelector } from "react-redux";
 import DataDisplayer from "../DataDisplayer";
 import { fetchEndpointData } from "../../utils/fetcher";
-import { Typography } from "@mui/material";
+import { Typography, Box } from "@mui/material";
+import { useGetSpellQuery } from "../../utils/api";
 
 const SpellsContent = ({ data }) => {
   return (
-    <>
+    <Typography variant="h5" component="div">
+      {data.name}
+    </Typography>
+  );
+};
+
+const ModalSpellContent = ({ data }) => {
+  return (
+    <Box>
       <Typography variant="h5" component="div">
         {data.name}
       </Typography>
       <Typography gutterBottom sx={{ color: "text.secondary", fontSize: 14 }}>
-        Incantation: {data.incantation ? data.incantation : "DEATHLY"}
+        Incantation: {data.incantation ? data.incantation : "Unknown"}
```

Comment:
if `data` doesn't contain such fields for any reason (for example, server error which is not predicted), and statement `data.incantation` will throw ReferenceError error, which is blocking one, and to avoid it you can use **?** operator ==> `data?.incantation`
With additional check, there won't be any blocking errors